src/components/message-input/index.test.tsx

```diff
@@ -96,6 +100,18 @@ describe('MessageInput', () => {
     expect(dropzone.find(ReplyCard).exists()).toBe(true);
   });
 
+  it('renders MessageAudioRecorder', function () {
+    const wrapper = subject({});
+    const dropzone = wrapper.find(Dropzone).shallow();
+
+    expect(dropzone.find(MessageAudioRecorder).exists()).toBe(false);
+
+    (wrapper.instance() as any).startMic();
+    const dropzone2 = wrapper.find(Dropzone).shallow();
```

Comment:
Are we able to reuse the `dropzone` reference from above? I think `dropzone.update()` or `dropzone.setProps({})` should trigger a re-render.